app/styles/main.scss

```diff
@@ -23,6 +23,10 @@ html, body {
   height: 100%;
 }
 
+.body-container {
+  min-height: calc(100vh - 317px);
```

Comment:
We'd add a media query to change the pixel subtraction at the same breakpoint that the footer height changes.